spec/models/bridge_spec.rb

```diff
@@ -1,56 +1,59 @@
 # frozen_string_literal: true
```

Comment:
Right now you are only testing perfect world scenarios. What about testing invalid data? Are you sure can handle invalid data?